resources/lang/jp/disposable.php

```diff
@@ -0,0 +1,80 @@
+<?php
```

Comment:
This file should **NOT** be included in this PR.

It is being shipped by a 3rd party addon already.